player/tests.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.auth import get_user_model
+from .models import Player, BallGame
+import pytest
+
+
+@pytest.fixture
+def user():
+    return get_user_model().objects.create_user(
+        username='testuser',
+        password='testpass'
+    )
+
+
+@pytest.fixture
+def player(user):
+    return Player.objects.create(
+        user=user,
+        birth_date='1990-01-01',
+        favorite_ball_game=BallGame.Soccer
+    )
+
+
+def test_create_player(player, user):
+    assert player.user == user
+    assert player.birth_date == '1990-01-01'
+    assert player.favorite_ball_game == BallGame.Soccer
+
+
+def test_update_player(player):
+    player.favorite_ball_game = BallGame.Basketball
+    player.save()
+    updated_player = Player.objects.get(pk=player.pk)
+    assert updated_player.favorite_ball_game == BallGame.Basketball
+
+
+def test_delete_player(player):
+    player.delete()
+    assert not Player.objects.filter(pk=player.pk).exists()
```

Comment:
What happens if you run this assertion before calling delete? I suspect that this player is not saved (not sure if it's the same case for the user as well). You can maybe add another fixture `saved_player` in which you'd save the player, and then have the tests use that fixture.

Reply:
In any case, you have to be convinced that the player is actually saved.